pages/bed-index-planner.tsx

```diff
@@ -248,7 +248,7 @@ export default function BedIndex({
           <Text mt="0 !important">
             {moment(activePrice.timestamp).format('DD MMMM, YYYY')}
           </Text>
-          <Box spacing={3} w="full" h={260}>
+          <Box w="full" h={260}>
```

Comment:
This was causing an error, but I want to make sure the design isn't drastically changed by removing spacing. This might need another update.

Reply:
My guess is that this prop was not actually doing anything, given that this is not a chakra-ui expected prop, nor is it any CSS attribute. The closest thing I suppose is "letter-spacing."

Are you able to post screenshots in the PR of before and after the change removing the spacing?

I also think that using a width of "full" is not conventional, and I wonder if that width is even doing anything.
https://developer.mozilla.org/en-US/docs/Web/CSS/width

In summary, the existing code is wrong on at least 2 of the three attribute props. I don't think your change is introducing any sort of regression, but please if you would just clarify what you see locally.

Reply:
> My guess is that this prop was not actually doing anything, given that this is not a chakra-ui expected prop, nor is it any CSS attribute. The closest thing I suppose is "letter-spacing."
> 
Spacing is special Chakra prop that allows you to quickly and evenly space.  https://chakra-ui.com/docs/layout/wrap#change-the-spacing for some quick example

> I also think that using a width of "full" is not conventional, and I wonder if that width is even doing anything. https://developer.mozilla.org/en-US/docs/Web/CSS/width
> 
Full is also a special Chakra UI thing that sets the width of the element to full screen size.

I'm having trouble with my local Ghost today, but Ill take a look as soon as I resolve the issue

Reply:
The spacing prop looks like it is expected on Wrap or SimpleGrid components but possibly the issue is that it is not expected on Box component.

Reply:
There were a few dependency issues, so I upgraded chakra `"@chakra-ui/icons": "^1.1.5", "@chakra-ui/react": "^1.8.1"`
I do believe spacing was a valid attribute in the older versions, but has now been deprecated. Looking at these screenshots, I don't see a noticeable different. The update was around the box/price chart. https://www.bankless.community/bed-index-planner

-----------------------------------------------------------------------------------------------
### Current Production with spacing
<img width="1355" alt="current-prod" src="https://user-images.githubusercontent.com/34350/153655481-8efac709-0554-4f34-bead-0316a1f6cd75.png">

-----------------------------------------------------------------------------------------------
### Preview with out spacing
<img width="1341" alt="preview-site" src="https://user-images.githubusercontent.com/34350/153655600-b7d476e5-6821-428d-aca7-18c2ee5ad8e2.png">